assignments/function-conversion.js

```diff
@@ -1,30 +1,31 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
-// console.log("Function was invoked!");
-// };
-// myFunction();
-
-// let anotherFunction = function (param) {
-//   return param;
-// };
-// anotherFunction("Example");
-
-// let add = function (param1, param2) {
-//   return param1 + param2;
-// };
-// add(1,2);
-
-// let subtract = function (param1, param2) {
-//   return param1 - param2;
-// };
-// subtract(1,2);
-
-
-// Stretch
-
-// exampleArray = [1,2,3,4];
-// const triple = exampleArray.map(function (num) {
-//   return num * 3;
-// });
-// console.log(triple);
\ No newline at end of file
+let myFunction = function (a) => console.log("Function was invoked!"
```

Comment:
The => implies that the code is a function so no need to declare it
```suggestion
let myFunction = (a) => console.log("Function was invoked!")
```

Reply:
The => implies that it is a function so there is no need to declare it. No parameter is require for the function to console.log the string that you provided it.